src/main/resources/definitions_with_links.txt

```diff
@@ -10,8 +10,12 @@ ComponentVersionView,origins,true,OriginView
 ComponentVersionView,vulnerabilities,true,VulnerabilityV2View
 ComponentVersionView,risk-profile,false,ComponentVersionRiskView
 
+CweView,sources
```

Comment:
The sources link seems to point to external sites. So I dont think a type would apply here